_data/sidebars/es_lb4_sidebar.yml

```diff
@@ -0,0 +1,441 @@
+## definition of `menuitem`===========
+# title: string
+# url: string
+# output: [ enum{web, pdf} ] (expressed as comma separated string in YAML)
+# children?: [ menuitem ]
+
+## top level property=================
+# entries: [ menuitem ]
+
+title: LoopBack 4
+url: /doc/es/lb4/index.html
+translated: true
+children:
+
+- title: 'Iniciando'
+  url: Getting-started.html
+  output: 'web, pdf'
+
```

Comment:
please add the `translated: false` all entries if they are not translated
[reference](https://github.com/strongloop/loopback.io/pull/841/files#diff-6a7c2646784a272c5705d77df7e4bc15R35) 😄 thanks

Reply:
Can you also ping us when you finish? We might not get notifications when you commit.

Reply:
@agnes512 Hi, I resolved can you check please ?